zcash_proofs/src/prover.rs

```diff
@@ -21,9 +21,9 @@ use crate::{default_params_folder, SAPLING_OUTPUT_NAME, SAPLING_SPEND_NAME};
 /// An implementation of [`TxProver`] using Sapling Spend and Output parameters from
 /// locally-accessible paths.
 pub struct LocalTxProver {
-    spend_params: Parameters<Bls12>,
-    spend_vk: PreparedVerifyingKey<Bls12>,
-    output_params: Parameters<Bls12>,
+    pub spend_params: Parameters<Bls12>,
+    pub spend_vk: PreparedVerifyingKey<Bls12>,
+    pub output_params: Parameters<Bls12>,
```

Comment:
Why `pub` is required here? I tried to remove it and lib compiled successfully.

Reply:
it's needed to do this <https://github.com/KomodoPlatform/komodo-defi-framework/blob/cf6891bc1794bba51609bd4190b4a1afba82cf3d/mm2src/coins/z_coin/storage/walletdb/wasm/mod.rs#L26-L35>. Additionally I think it will be needed when implementing zcash_params for WASM

Reply:
just figured out that there's a method named `from_bytes` that takes in `spend_buf` and `output_buf` to create `LocalTxProver`...so I'll revert

Reply:
@borngraced It is still there